examples/browser-flags-example/popup.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Browser Flags Example</title>
+  <style>
+    body {
+      font-family: Arial, sans-serif;
+      width: 300px;
+      padding: 15px;
+    }
+    h1 {
+      font-size: 18px;
+      color: #4285f4;
+    }
+    button {
+      background-color: #4285f4;
+      color: white;
+      border: none;
+      padding: 8px 16px;
+      border-radius: 4px;
+      cursor: pointer;
+      margin: 5px 0;
+    }
+    button:hover {
+      background-color: #3367d6;
+    }
+    .container {
+      display: flex;
+      flex-direction: column;
+      gap: 10px;
+    }
+    .feature {
+      border: 1px solid #ddd;
+      padding: 10px;
+      border-radius: 4px;
+    }
+    .scrollable {
+      height: 100px;
+      overflow-y: auto;
+      border: 1px solid #ddd;
+      padding: 10px;
+      margin: 10px 0;
+    }
+  </style>
+</head>
+<body>
+  <h1>Browser Flags Example</h1>
+  <p>This example demonstrates features that require custom browser flags:</p>
+  
+  <div class="container">
+    <!-- Audio Feature Demo -->
+    <div class="feature">
+      <h2>Audio Feature</h2>
+      <p>Test if audio works (requires <code>--mute-audio</code> to be disabled)</p>
+      <button id="playAudio">Play Sound</button>
+      <audio id="testAudio" src="data:audio/wav;base64,UklGRl9vT19XQVZFZm10IBAAAAABAAEAQB8AAEAfAAABAAgAZGF0YU..." preload="none"></audio>
```

Comment:
I am entirely unsure if this works - I actually tested by opening youtube in the browser.

Reply:
it works :+1: